src/components/ButtonBox/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Button from "../Button/index.jsx";
+import styles from "./styles/styles.module.scss";
+
+const ButtonBox = ({
+  PreviousButtonType,
+  PreviousButtonText,
+  previousPath,
+  nextButtonType,
+  nextButtonText,
+  nextPath,
+}) => {
+  let buttonBox = (
+    <div className={styles.one_button_box}>
```

Comment:
Please change `styles.one_button_box` to something more specific.

Reply:
done